### PR TITLE
Portfolio - mobile menu button

### PR DESCRIPTION
**Pull request by Kingno3l**
- [ ] 1. Used basic JavaScript syntax.
- [ ] 2. Used JavaScript to manipulate DOM elements.
- [ ] 3. Used JavaScript events.

The following features were implemented:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Kingno3l, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a branch conflict. please resolve it and submit it again. 
### Error
Screenshot 2023-01-04 151822


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -25,15 +25,20 @@
...
+                    
+                    <!-- <div class="hamburger_icon mobile"> <img src="img/hamburger-menu-icon.png" width="20"> -->
+                    <!-- </div> -->
+                    
                         <ul class="navlink">
-                            <li>Portfolio</li>
-                            <li>About</li>
-                            <li>Contact</li>
+                            <li> <a href="#portfolioid"> Portfolio </a> </li>
+                            <li> <a href="#aboutid"> About </a> </li>
```
Comment (amejid) [inline index.html:39]: - [ ] Kindly check if the `About` link is scrolling to the corresponding section when clicked. Currently, it is not doing anything when clicking on it. Please update it accordingly. 🤝️

File: mystyle.css [hunk 2]
```diff
@@ -625,6 +677,34 @@ For small, less significant text and placeholders.
...
+    transition: 0.3s;
+  }
+
+  .navlink {
+    position: unset;
+    height: auto;
+    display: flex;
+    width: auto;
+    background-color: transparent;
+  }
```
Comment (amejid) [inline mystyle.css]: - [ ] One of the objectives of this project is to understand how to parse a Figma design precisely. Kindly update the `font-size` and position of the nav links. 
- [ ] When the nav menu is opened, it should fill the entire width of the screen. Please update it according to the Figma design. 🧐️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Kingno3l, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Descriptive PR :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @Kingno3l , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` and `title`
## Optional suggestions
- [ ] For the hamburger background you may consider increasing the `filter:blur(8px)` to make it more blurred :+1: 


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kingno3l** [comment]
Great!
Thank you!!!

**Comment by ichala** [comment]
Hi @Kingno3l   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Project Already approved ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**